lib/src/client.dart

```diff
@@ -2134,6 +2194,13 @@ class Client extends MatrixApi {
       _discoveryDataLoading = database.getWellKnown().then((data) {
         _wellKnown = data;
       });
+      _oidcAuthMetadataLoading = database.getOidcAuthMetadata().then((data) {
```

Comment:
I doubt that this really needs to be waited for. Can't we just make it part of the `Database.getClient` call?

Reply:
This is still not resolved